README-QA.md

```
@@ -0,0 +1,108 @@
+- `deokhugam_db` 데이터베이스가 생성되어 있을 것
+
+```bash
+# 예시 — 데이터베이스 생성
+createdb deokhugam_db
+```
+
+#### 2. `.env` 파일
+
+프로젝트 루트의 `.env`에 connection 정보를 작성한다 (이미 있다면 그대로).
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)